components/apps.js

```diff
@@ -535,7 +550,8 @@ SteamUser.prototype._getLicenseInfo = async function() {
 	}
 
 	try {
-		await this.getProductInfo(appids, [], false, undefined, PICSRequestType.PackageContents);
+		await this.getProductInfo(appids, [], true, undefined, PICSRequestType.PackageContents);
+        this.picsCache.ownershipModified = Date.now();
```

Comment:
Previously, `this.picsCache.packages` was checked to see if `appOwnershipCached` event was emitted. However, this is not a good check, since `this.picsCache.packages` always returns `true`, [as it is initialized as an empty set](https://github.com/DoctorMcKay/node-steam-user/blob/master/index.js#L28).
Therefore, I introduce this property to `this.picsCache`. Admittedly, I could have fixed it by checking `Object.keys(this.picsCache.packages).length > 0` instead, but using this property is faster and more informative.